demo/ksnav_app_linux/Dockerfile_ksnav

```diff
@@ -0,0 +1,123 @@
+#BUILDER 
+FROM quay.io/centos/centos:stream9 AS builder
+
+# build ARGs
+ARG _RADARE2_GIT_REPO="https://github.com/radareorg/radare2.git"
+ENV RADARE2_GIT_REPO=$_RADARE2_GIT_REPO
+# Radare2 constitutes a noteworthy dependency. 
+# However, it's a software that undergoes consistent changes across commits. 
+# In my experience, even within the same minor version, significant alterations can be observed. 
+# Despite ks-nav not utilizing any uncommon features of Radare2, I prefer to adhere to the most thoroughly tested commit.
+#ARG _RADARE2_GIT_COMMIT=4a07098809c71c21515d0f862ac4ca044ae95e95
+#ENV RADARE2_GIT_COMMIT=$_RADARE2_GIT_COMMIT
+ARG _KS_NAV_GIT_REPO="https://github.com/elisa-tech/ks-nav"
+ENV KS_NAV_GIT_REPO=$_KS_NAV_GIT_REPO
+ARG _RADARE2_GIT_REPO_TAG="5.9.0"
+ENV RADARE2_GIT_REPO_BRANCH=$_RADARE2_GIT_REPO_TAG
+
+# install build dependecies
+RUN dnf install -y autoconf automake gcc gcc-c++ git make patch
+RUN dnf install -y golang
+RUN dnf install -y kernel-devel
+RUN dnf install -y bzip2
+
+# build app
+WORKDIR /build
+
+# Build radare2
+RUN git clone -b ${RADARE2_GIT_REPO_BRANCH} ${RADARE2_GIT_REPO} /build/radare2 && cd /build/radare2 
+RUN cd /build/radare2 && ./configure && make && make install
+
+# Build ks-nav in the WORKDIR folder
+RUN git clone ${KS_NAV_GIT_REPO} /build/ksnav
+#COPY ks-nav /build/ksnav
+COPY ksnav_kdb.json /build/. 
```

Comment:
Need to clarify why these were missing and what they fix

Reply:
Each of these git commits that update existing files needs to be a separate commit with good descriptions detailing the reason for the change.